templates.json

```diff
@@ -595,8 +595,8 @@
     ]
   },
   {
-    "identifier": "discount_function_metafields",
-    "name": "Discount Function Metafields - UI Extension",
+    "identifier": "discount_details_function_settings",
+    "name": "Discount Details Settings - UI Extension",
     "defaultName": "Discount Function Metafields UI",
```

Comment:
This one should be `Discount Function Settings` - I'd remove the UI part unless the name becomes less than 30 characters when snake cased.